hw/dv/tools/vcs/cover.cfg

```diff
@@ -19,3 +19,14 @@ begin tgl
   +module prim_prince
   +module prim_lfsr
 end
+
+begin assert
+  // These three assertions in prim_lc_sync check when `lc_ctrl_pkg::lc_tx_e` input is neither `On`
+  // or `Off`, it is interrupted to the correct `On` or `Off` after one clock cycle. This behavior
+  // is implemented outside of IP level design thus these assertions are not covered in IP level
+  // testbenchs.
+  // TODO: check these assertions in top-level or FPV.
+  -assert PrimLcSyncCheckTransients_A
+  -assert PrimLcSyncCheckTransients0_A
+  -assert PrimLcSyncCheckTransients1_A
```

Comment:
I tried using `PrimLcSyncCheckTransients*_A` but get a compile error that might related to memory usage. So I listed all these assertions instead.